src/index.html

```diff
@@ -9,6 +9,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="navbar">
+      <a class="logo nav__link" href="#">
+        <img src="./images/logo.png" alt="logo">
```

Comment:
- alt attribute should describe the image content
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/ab080d33-3414-425b-abd4-cdbc9433643b)